app/common/dispatcher/dispatchAction.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function (name, action) {
```

Comment:
also wanted to remove the appStore dependency for actions because it causes a lot of circular dependency issues